libs/sdk-ui-charts/src/charts/headline/internal/interfaces/BaseHeadlines.ts

```diff
@@ -7,9 +7,16 @@ export type BaseHeadlineDataItemComponentType = ComponentType<
     IBaseHeadlineDataItemProps & IWithDrillableItemProps & IWithTitleProps
 >;
 
+export enum ItemStatus {
+    NEGATIVE_VALUE = "negative",
+    EQUALS_VALUE = "equals",
+    POSITIVE_VALUE = "positive",
+}
+
 export interface IBaseHeadlineItem {
     data: IHeadlineDataItem;
     baseHeadlineDataItemComponent: BaseHeadlineDataItemComponentType;
+    itemStatus?: ItemStatus;
```

Comment:
I would call it just `status`. It's already scoped by `IBaseHeadlineItem` which contains word "item". Maybe it could be called differently. How would you describe this prop? What it does to relation to the value you put there? I'm struggle with finding better alternative.

Reply:
Thank you for your comment! 
I have updated the property and type to `evaluationType: EvaluationType` to provide a clearer description of its relationship with the value (with 3 values: `positive_value`, `negative_value`, `equals_value`)

Reply:
Good job with the naming. I like it now! It make much more sense without context 💪